Dockerfile

```diff
@@ -22,7 +22,6 @@ COPY . .
 RUN go test ./...  && \
     go install \
         -ldflags "-w -X 'github.com/Pix4D/cogito/resource.buildinfo=$BUILD_INFO'" \
-        ./cmd/check \
```

Comment:
we don't build the check executable any more, instead we symlink it to the cogito executable (see line 41)